pkg/api/pod/util.go

```diff
@@ -331,6 +331,25 @@ func usesHugePagesInProjectedEnv(item api.Container) bool {
 	return false
 }
 
+// hasSysctlsWithSlashNames return value is determined by the following:
+// 1.use strict validation for Create requests, and for Update requests where the existing object's sysctl does not contain a forward slash, return false
+// 2.use relaxed validation for Update requests where the existing object's sysctl contains a forward slash, return true
+func hasSysctlsWithSlashNames(podSpec *api.PodSpec) bool {
+	if podSpec.SecurityContext == nil {
+		return false
+	}
+	securityContext := podSpec.SecurityContext
+	if len(securityContext.Sysctls) != 0 {
```

Comment:
can drop the length check to simplify; the range will be a no-op on a zero length Sysctls list